code/modules/SCP/SCPs/SCP-049.dm

```diff
@@ -61,19 +60,54 @@ GLOBAL_LIST_EMPTY(scp049_1s)
 		/mob/living/carbon/human/scp049/proc/cure_action
 	)
 
+
+	var/datum/spell/spl = new /datum/spell/targeted/curepestillence
+	add_spell(spl)
+	mind.learned_spells += spl
+	spl = new /datum/spell/aimed/stopheart
+	add_spell(spl)
+	mind.learned_spells += spl
+
 /mob/living/carbon/human/scp049/Destroy()
 	pestilence_images = null
 	attempted_surgery_on = null
 	target = null
 	GLOB.scp049s -= src
 	return ..()
 
+/mob/living/carbon/human/scp049/examinate(var/mob/living/carbon/human/target)
+	. = ..()
+	if(prob(1) && !(target in infected_players) && is_valid_curing_target(target) && world.time >= pestillence_cooldown)
```

Comment:
That means that 049 can spam-examine people and eventually they'll be "infected".

Reply:
Fixed already, haven't pushed the changes yet. Added a cooldown of 5 minutes. Might reconsider it entirely though.